config/users/users.yaml

```diff
@@ -0,0 +1,50 @@
+parameters:
+    users:
+        vader:
+            name: Darth Vader
+            email: d.vader@empire.gov
+            givenName: Anakin
+            familyName: Skywalker
+            middleName: ~
+            locale: en
+            picture: https://cdn3.iconfinder.com/data/icons/star-wars-color/216/darth-vader-512.png
+        obiwan:
+            name: Obiwan Kenobi
+            email: o.kenobi@jedi.com
+            givenName: Obiwan
+            familyName: Kenobi
+            middleName: ~
+            locale: en
+            picture: https://cdn3.iconfinder.com/data/icons/star-wars-color/216/obiwan-kenobi-512.png
+        maul:
+            name: Darth Maul
+            email: d.maul@empire.gov
+            givenName: ???
+            familyName: ???
+            middleName: ~
+            locale: en
+            picture: https://cdn3.iconfinder.com/data/icons/star-wars-color/181/darth-maul-512.png
+        chewbacca:
+            name: Chewbacca
+            email: chewie@rebels.com
+            givenName: ???
+            familyName: ???
+            middleName: Chewie
+            locale: en
+            picture: https://cdn3.iconfinder.com/data/icons/star-wars-color/229/chewbacca-512.png
+        sidious:
+            name: Darth Sidious
+            email: d.sidious@empire.gov
+            givenName: Sheev
+            familyName: Plapatine
```

Comment:
Isn't it Palpatine ?
-> blocking issue here :smile:

Reply:
:) fixed